src/monopoly/monopoly_token.js

```diff
@@ -0,0 +1,12 @@
+var inherits = require('util').inherits;
+
+var TableTop = require('../../tabletop/tabletop');
+
+function MonopolyToken(cssClass) {
+    TableTop.Token.call(this);
+    this.cssClass = cssClass;
```

Comment:
cssClass will be an optional parameter for all tokens. There will be a default styling for your tokens, but to add additional styling you can add additional css options. There's a better implementation we will eventually switch to where we can define the styles by js class name in an object in the views. Original implementation definitely causes the view logic to infringe on the model too much.